control_panel_api/settings/base.py

```diff
@@ -1,6 +1,15 @@
 import os
 
-import boto3
+
+def is_enabled(value):
+    return str(value).lower() not in ('n', 'no', 'off', 'false', '0')
```

Comment:
I wonder if we should keep it simple and consider a flag enabled only when equal to string `'true'`.

```python
def is_enabled(value):
  return str(value).lower() == 'true'
```

Reply:
That seems unnecessary as the code is already simple and is more flexible that what you suggest.